### DISCUSSION TITLE
Kia Branding Update

### DISCUSSION THREAD

**Discussion by PeteCooney** [Hyundai-Kia-Connect/kia_uvo#526] [General]
Kia have recently announced that they are dropping the "Kia UVO" brand in favor of standardising "Kia Connect" in all territories. 
Appreciate that this is mainly a name change and doesn't impact upon functionality in any way but would be great if this brilliant integration reflected the correct branding (although, I appreciate that changing the name risks breaking things for users so maybe something for the next major release?)

**Comment by cdnninja** [comment]
I do believe this would break some things as we use kia_uvo as the integration title.  When I get a chance I will create PR to re-title the items that aren't impacted by this.
